### ISSUE TITLE
[Feature]: Standard Push Notification Setup missing for AppDelegate.mm/Objective-C

### ISSUE THREAD

**Issue by lucasGabrielDeAA** [braze-inc/braze-react-native-sdk#228]
### What problem are you facing?

I am integrating In-App Messages and Push Notifications within my react-native app, but I am no able to find the objective-C version of the AppDelegate's file to setup push notifications. I am following this guide [here](https://braze-inc.github.io/braze-swift-sdk/tutorials/braze/b1-standard-push-notifications/).
I already tried looking on the [official documentation](https://braze-inc.github.io/braze-swift-sdk/documentation/brazekit/braze/configuration-swift.class/push-swift.class/automation-swift.property/) for iOS's SDK properties, but it didn't helped.

I need the object-C version for the following lines, since I cannot find any version in the documentation.
```
   configuration.push.automation = true
   // Manually enable or disable individual settings by overriding properties of `automation`. 
   configuration.push.automation.requestAuthorizationAtLaunch = false
```
Is there any way that you guys can help me? Or maybe anyone else here faced the same issue?
Thanks for your support, guys!

### Workarounds

No

**Comment by jerielng** [comment]
Hi @lucasGabrielDeAA, these push automation features are in the underlying Braze Swift SDK, which were released recently in version [`6.5.0`](https://github.com/braze-inc/braze-swift-sdk/releases/tag/6.5.0). At the moment, our React Native SDK hasn't been updated to this version yet (currently on `6.4.0`), but we plan to update our native iOS bindings in the next version.

At the moment, we don't have a timeline for this, but we will keep you updated here when it's been released. Thanks for your patience!

**Comment by lucasGabrielDeAA** [comment]
Hi @jerielng thanks for you reply and support! 😄

**Comment by jackster-humi** [comment]
Hi - what's the timeline for bumping native dependencies?

Our team have bridged to your native iOS SDK at the moment, but we prefer implementing your react-native SDK.

**Comment by jerielng** [comment]
Hi @jackster-humi, we're wrapping up some internal testing and looking to prepare the next release over the next 2-3 weeks.

**Comment by hokstuff** [comment]
Hi @lucasGabrielDeAA @jackster-humi,

We have just released [React Native SDK version 7.0.0](https://github.com/braze-inc/braze-react-native-sdk/releases/tag/7.0.0) which updates the iOS bindings, so you will be able to use the latest iOS native layer updates.

Thanks
